src/coreclr/vm/jitinterface.cpp

```diff
@@ -454,15 +454,15 @@ CEEInfo::ConvToJitSig(
     SigTypeContext typeContext;
 
     uint32_t sigRetFlags = 0;
-    if (pContextMD)
+    if (context.methodContext)
     {
-        SigTypeContext::InitTypeContext(pContextMD, contextType, &typeContext);
-        if (pContextMD->ShouldSuppressGCTransition())
+        SigTypeContext::InitTypeContext(context.methodContext, context.typeContext, &typeContext);
+        if (context.methodContext->ShouldSuppressGCTransition())
```

Comment:
The suppress GC transition flag is really part of the unmanaged calling convention. Should it be moved to the API you are adding, e.g. as a new `out bool` argument?